scripts/release.sh

```diff
@@ -36,13 +36,14 @@ require('fs').writeFileSync('docs/manifest.json', JSON.stringify(file, null, 2)
 set +e
 git add docs/manifest.json
 set -e
-git commit -m "Update docs/manifest.json for ${TRAVIS_TAG} [ci skip]"
-git status
 if [[ -n "$(git status --porcelain)" ]]; then
+  git commit -m "Update docs/manifest.json for ${TRAVIS_TAG} [ci skip]"
+  git status
   git push https://${GH_OAUTH_TOKEN}@github.com/${GH_OWNER}/${GH_PROJECT_NAME} master
 else
   echo "docs/manifest.json already includes the new version. Skipping commit."
 fi
+cd ../
```

Comment:
We have to back out of the `master` branch, because `npm run docs` wasn't run there.